docs/source/environments/index.rst

```diff
@@ -20,24 +20,26 @@ environment!
     All environments offer a default no-op action via :code:`env.action_space.no_op()`
     and a random action via :code:`env.action_space.sample()`.
 
+
 .. include:: handlers.rst
     :end-before: inclusion-marker-do-not-remove
 
+
 Basic Environments
 =======================================
 
 .. warning::
 
-    The following basic environments are NOT part of the 2020 MineRL Competition! Feel free to use them for exploration
-    but agents may only be trained on `MineRL competition environments`_!
+    The following basic environments are NOT part of the 2021 MineRL Competition! Feel free to use them for exploration
```

Comment:
Maybe any of the 2021 MineRL competitions to include basalt?

Reply:
yeah, could change Competition! to `Competitions!` here.

I think the ~"You can't do this in Diamond (omitting any such rule for BASALT)" statement here is fine because as I understand it we are pretty lenient in BASALT with training methods (you can somehow train your agent on TreeChop if you wanted to?). @rohinmshah

Reply:
I updated to the following. Let me know if I'm misunderstanding something related to the rules of either competition and we can adjust accordingly

![image](https://user-images.githubusercontent.com/1750835/121276114-e187a900-c882-11eb-8f3e-f68bf7692ef9.png)

Reply:
Oh - good point perhaps then we should only say that these are not a part of MineRL Diamond

Reply:
Actually I don't think we should allow participants to train on TreeChop -- that's extracting an additional "did you chop a tree" reward that they shouldn't have access to.

(Doesn't matter very much for TreeChop, since they could make that reward themselves based on their inventory, but the general principle would be, don't use other environments since they might extract additional disallowed info from the Minecraft simulator.)

I might change it to:

> Feel free to use them for exploration but note that MineRL competition agents may only be trained on the corresponding competition environments.

Reply:
yeah, that sounds good @rohinmshah , I'll use that instead (I think that this wording covers the intent of the old message too :) )